### PR TITLE
Added options for oc adm release info

### PR DESCRIPTION
**Pull request by pdhamdhe**
@xiaojiey Could you please help to review _oc adm release info_ command options ? Thanks

### CODE REVIEW

File: lib/rules/cli/4.1.yaml [hunk 1]
```diff
@@ -1291,6 +1291,8 @@
 :oadm_release_info:
   :cmd: oc adm release info
   :options:
+    :image-for: --image-for=<value>
```
Comment (pruan-rht) [inline lib/rules/cli/4.1.yaml:1294]: @pdhamdhe ruby naming convention for names are usually `_` instead of `-`

Comment (pdhamdhe) [reply lib/rules/cli/4.1.yaml]: Done. Thanks

**Comment by pdhamdhe** [comment]
@pruan-rht @liangxia Please help to review. Thanks!

**Pull request merged**